### ISSUE TITLE
wrk under aarch64 (ARM64) fails

### ISSUE THREAD

**Issue by tobbik** [termux/termux-packages#5291]
**Running wrk on an ARM-64 platform(ASUS C101) gives _bus error_**
The error could be reproduced on a Raspberry pi4 under ArchLinuxARM64. It is descriped upstream and has an existing pull request upstream: https://github.com/wg/wrk/pull/358/commits/e2ab808cd1de82d3622f70234e981ee941db7122

**Steps to reproduce**
Run wrk on a ARM-64 hardware against a known server. On my C101 `wrk -t 4 -c 10 -d 5 http://www.google.com` fails repeatedly.

**Expected behavior**
It should run without the bus error.

**Additional information**
I used below patch (copied from the pull request) and compiled it on ARM-64, ARM-32 and x86-64 and it worked without problems. it wouldn't allow to add this patch as attachement so I c'np it here

```
diff -ur wrk-4.1.0-orig/src/zmalloc.c wrk-4.1.0/src/zmalloc.c
--- wrk-4.1.0-orig/src/zmalloc.c	2020-05-07 01:47:50.562052028 +0000
+++ wrk-4.1.0/src/zmalloc.c	2020-05-07 01:48:12.281671231 +0000
@@ -45,13 +45,19 @@
 #include "zmalloc.h"
 #include "atomicvar.h"
 
+#ifdef _LP64
+#define ALIGMENT (16)
+#else
+#define ALIGMENT (8)
+#endif
+#define ROUND_UP(n,r) (((n + r - 1) / r ) * r)
 #ifdef HAVE_MALLOC_SIZE
 #define PREFIX_SIZE (0)
 #else
 #if defined(__sun) || defined(__sparc) || defined(__sparc__)
-#define PREFIX_SIZE (sizeof(long long))
+#define PREFIX_SIZE (ROUND_UP(sizeof(long long), ALIGMENT))
 #else
-#define PREFIX_SIZE (sizeof(size_t))
+#define PREFIX_SIZE (ROUND_UP(sizeof(size_t), ALIGMENT))
 #endif
 #endif
 
```

**Comment by xeffyr** [comment]
Should be fixed in 4.1.0-2. Try `pkg upgrade`.

**Comment by tobbik** [comment]
Thank you, confirmed to work on ASUS C101 (armv8l).
